CONTRIBUTING.md

```diff
@@ -1,3 +1,20 @@
+## Steps for Contributing
+1. Create a new branch off of up-to-date master
```

Comment:
Go ahead and make each of these lines start with `1.` to turn it into an ordered list.